Dockerfile

```diff
@@ -71,44 +54,21 @@ RUN \
   if test -n "$CABAL_VERSION"; then \
     ghcup install cabal "$CABAL_VERSION" --set; \
     cabal --version; \
-  fi
-
-# Configure Cabal.
-
-ARG CABAL_STORE=/cabal-store
```

Comment:
Instead of using the non-standard `/cabal-store`, now we'll be using `~/.local/state/cabal/store`. Arguably `/cabal-store` is more portable since it doesn't assume your username is `vscode`. But if you were a different user, you'd have to set Cabal's store directory anyway, and setting it to `/home/vscode/.local/state/cabal/store` isn't any harder than setting it to `/cabal-store`.

Reply:
Everyone will have to update their Docker volumes, regardless of which editor they use.